weaver/wps_restapi/processes/processes.py

```diff
@@ -315,7 +315,7 @@ def submit_job_handler(request, service_url, is_workflow=False, visibility=None)
     except Exception as ex:
         raise HTTPBadRequest("Invalid JSON body cannot be decoded for job submission. [{}]".format(ex))
     try:
-        json_body = sd.Execute().deserialize(json_body)
+        sd.Execute().deserialize(json_body)
```

Comment:
This causes optional values from the input, but required later below, not to be filled in the body with defaults.